x-pack/plugin/security/src/test/java/org/elasticsearch/xpack/security/authz/AuthorizedIndicesTests.java

```diff
@@ -105,36 +108,43 @@ public void testAuthorizedIndicesUserWithSomeRoles() {
             future
         );
         Role roles = future.actionGet();
-        Set<String> list = RBACEngine.resolveAuthorizedIndicesFromRole(
+        AuthorizedIndices list = RBACEngine.resolveAuthorizedIndicesFromRole(
```

Comment:
Nit: Can we renamed the variable to `authorizedIndices`? The existing name `list` is rather confusing after this refactor. The same comment applies to two other methods (`testDataStreamsAreNotIncludedInAuthorizedIndices`, `testDataStreamsAreIncludedInAuthorizedIndices`) in this test class as well.

Reply:
Pushed [d8953b2](https://github.com/elastic/elasticsearch/pull/91618/commits/d8953b2449110a85d65af30d231ed556c57e9c00)